lib/langchainrb_rails/generators/langchainrb_rails/pgvector_generator.rb

```diff
@@ -24,7 +24,7 @@ def migration_version
       end
 
       def add_to_model
-        inject_into_class "app/models/#{model_name.downcase}.rb", model_name do
+        inject_into_class "app/models/#{model_name.underscore}.rb", model_name do
```

Comment:
Should this be `.tableize` as well? I'm wondering if that's how Rails does it as well.

Reply:
`.tableize` pluralizes the name which we do not want.